firefox_puppeteer/ui/menu.py

```diff
@@ -6,10 +6,10 @@
 from marionette_driver.errors import NoSuchElementException
 
 from firefox_puppeteer import DOMElement
-from firefox_puppeteer.base import BaseLib
+from firefox_puppeteer.ui_base_lib import UIBaseLib
 
 
-class MenuBar(BaseLib):
+class MenuBar(UIBaseLib):
```

Comment:
This should not only stand for menubar like elements but a menu in general. The `menubar` property of the browser window is just a specific instance of a `Menu` class.